### PR TITLE
Add Music MiniPlayer v1.8

### PR DESCRIPTION
**Pull request by dannydorazio**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ x ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ x ] `brew audit --cask --online <cask>` is error-free.
- [ x ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ x ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ x ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ x ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ x ] `brew audit --new-cask <cask>` worked successfully.
- [ x ] `brew install --cask <cask>` worked successfully.
- [ x ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by razvanazamfirei** [comment]
Hi, @dannydorazio! Thank you for submitting this! Could you add a livecheck stanza? There's an appcast here: https://github.com/marioaguzman/marioaguzman.github.io/blob/master/musicminiplayer/updater/musicminiplayerappcast.xml

File: Casks/music-miniplayer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask "music-miniplayer" do
+  version "1.8"
+  sha256 :no_check
+
+  url "https://marioaguzman.github.io/musicminiplayer/updater/Music%20MiniPlayer.zip"
+  name "Music MiniPlayer for macOS"
```
Comment (razvanazamfirei) [inline Casks/music-miniplayer.rb:6]:
```suggestion
  name "Music MiniPlayer"
```

File: Casks/music-miniplayer.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  desc "Replica of the classic iTunes MiniPlayer to control Apple Music"
```
Comment (razvanazamfirei) [inline Casks/music-miniplayer.rb:7]:
```suggestion
  desc "Replica of the iTunes MiniPlayer"
```

File: Casks/music-miniplayer.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  homepage "https://marioaguzman.github.io/musicminiplayer/"
+
+  auto_updates true
+
+  app "Music MiniPlayer.app"
+
+  zap trash: [
+    "~/Library/Preferences/com.marioaguzman.Music-MiniPlayer.plist",
+    "~/Library/Saved Application State/com.marioaguzman.Music-MiniPlayer.savedState",
+    "~/Library/Caches/com.marioaguzman.Music-MiniPlayer",
```
Comment (razvanazamfirei) [inline Casks/music-miniplayer.rb:17]:
```suggestion
    "~/Library/Caches/com.marioaguzman.Music-MiniPlayer",
    "~/Library/Preferences/com.marioaguzman.Music-MiniPlayer.plist",
    "~/Library/Saved Application State/com.marioaguzman.Music-MiniPlayer.savedState",
```

**Review by razvanazamfirei — DISMISSED** [review]

**Comment by dannydorazio** [comment]
> Hi, @dannydorazio! Thank you for submitting this! Could you add a livecheck stanza? There's an appcast here: https://github.com/marioaguzman/marioaguzman.github.io/blob/master/musicminiplayer/updater/musicminiplayerappcast.xml

Hey @razvanazamfirei, I added the livecheck stanza per your request in accordance with the Cask Cookbook [at least I think I did it right], and I committed the other modifications to the ruby file that you notated.  I appreciate your patience— I have never contributed to Homebrew [or really any public repo] before so I am doing this to try to learn the process and add some cool software into HB.  Please let me know if there is anything else I need to change.  Thanks!

**Review by bevanjkay — APPROVED** [review]
Thanks @dannydorazio - great first submission to `homebrew-cask`. Thank you 🎉

**Pull request merged**